<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1767</width>
    <height>1294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>robotshop</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/robot.png</normaloff>:/robot.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabBar::tab{width:200}
QTabBar::tab{height:40}
QTabBar::tab{font: 75 14pt &quot;微软雅黑&quot;}
QTabBar::tab:selected{font: 14pt &quot;华文行楷&quot;;color:#55aaff}




</string>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>   地图与控制   </string>
       </attribute>
       <widget class="QCustomPlot" name="widget" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1721</width>
          <height>1101</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <widget class="QStackedWidget" name="interaction">
         <property name="geometry">
          <rect>
           <x>1320</x>
           <y>10</y>
           <width>391</width>
           <height>141</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QStackedWidget{
border-radius:12px;
border: 2px solid  rgb(118, 238, 176);
background-color: rgb(255, 255, 255);
}
QPushButton{
color: rgb(255, 255, 255);
font: 75 12pt &quot;等线&quot;;
background-color:  rgb(118, 238, 176);
border-radius:8px !important;
}
QLineEdit{
border: 1px solid rgb(118, 238, 176);
border-radius: 5px !important;
background-color: rgb(255, 255, 255);
}
QLineEdit::placeholder {
    color: red;
    font-style: italic;
}




</string>
         </property>
         <property name="currentIndex">
          <number>4</number>
         </property>
         <widget class="QWidget" name="page">
          <widget class="QLineEdit" name="lineEdit">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>10</y>
             <width>131</width>
             <height>31</height>
            </rect>
           </property>
           <property name="placeholderText">
            <string>markname</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton">
           <property name="geometry">
            <rect>
             <x>190</x>
             <y>10</y>
             <width>111</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>删除站点</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_7">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>50</y>
             <width>61</width>
             <height>31</height>
            </rect>
           </property>
           <property name="placeholderText">
            <string>markstart</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_8">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>50</y>
             <width>61</width>
             <height>31</height>
            </rect>
           </property>
           <property name="placeholderText">
            <string>markend</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_4">
           <property name="geometry">
            <rect>
             <x>190</x>
             <y>50</y>
             <width>111</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>删除路径</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="page_2">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLineEdit" name="lineEdit_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="placeholderText">
                <string>x</string>
               </property>
               <property name="cursorMoveStyle">
                <enum>Qt::LogicalMoveStyle</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_3">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">placeholderText{
fontFamily: 12 italic 17pt &quot;Cascadia Code ExtraLight&quot;;
background-color: rgb(170, 170, 255);
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="placeholderText">
                <string>y</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_4">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="placeholderText">
                <string>yaw</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>增加站点</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLineEdit" name="lineEdit_5">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="toolTipDuration">
                <number>-5</number>
               </property>
               <property name="placeholderText">
                <string>markstart</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_6">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="placeholderText">
                <string>markend</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_3">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>增加路径</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLineEdit" name="lineEdit_9">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_10">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_11">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_12">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_5">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>删除障碍物</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_3">
          <widget class="QLineEdit" name="clamp_angle">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>20</y>
             <width>61</width>
             <height>41</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="clamp_distance">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>20</y>
             <width>71</width>
             <height>41</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="clamp_height">
           <property name="geometry">
            <rect>
             <x>190</x>
             <y>20</y>
             <width>71</width>
             <height>41</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="clamp_control">
           <property name="geometry">
            <rect>
             <x>280</x>
             <y>20</y>
             <width>80</width>
             <height>41</height>
            </rect>
           </property>
           <property name="text">
            <string>抓取</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="page_4">
          <widget class="QPushButton" name="pushButton_7">
           <property name="geometry">
            <rect>
             <x>260</x>
             <y>10</y>
             <width>101</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>导航</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_13">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>10</y>
             <width>51</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_14">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>10</y>
             <width>51</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_15">
           <property name="geometry">
            <rect>
             <x>190</x>
             <y>10</y>
             <width>51</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_16">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>50</y>
             <width>51</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_17">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>50</y>
             <width>51</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_18">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>50</y>
             <width>51</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_19">
           <property name="geometry">
            <rect>
             <x>190</x>
             <y>50</y>
             <width>51</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_8">
           <property name="geometry">
            <rect>
             <x>260</x>
             <y>50</y>
             <width>101</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>站点导航</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="page_5">
          <widget class="QLineEdit" name="IP">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>10</y>
             <width>191</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>127.0.0.1</string>
           </property>
           <property name="placeholderText">
            <string>IP地址</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="PORT">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>50</y>
             <width>191</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>8888</string>
           </property>
           <property name="placeholderText">
            <string>端口号</string>
           </property>
          </widget>
          <widget class="QPushButton" name="connect">
           <property name="geometry">
            <rect>
             <x>230</x>
             <y>30</y>
             <width>111</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>连接</string>
           </property>
          </widget>
         </widget>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>1090</y>
          <width>1721</width>
          <height>81</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox
{
border: 2px solid green;
border-radius:8px;
margin-top:2ex;
font:bold 22px &quot;Segoe Print&quot;;
}
QGroupBox::title
{
subcontrol-origin:margin;
subcontrol-position:top center;
padding:0 3px;
}
QPushButton{
background-color:transparent;
color:white;
border-radus:10px;
border-style:outset;
}
QPushButton:hover {
background-color: rgb(100, 100, 100);
}

</string>
        </property>
        <property name="title">
         <string>Map toolbox</string>
        </property>
        <widget class="QPushButton" name="move_map">
         <property name="geometry">
          <rect>
           <x>830</x>
           <y>40</y>
           <width>41</width>
           <height>29</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>91</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/res/mapmove.png</normaloff>:/res/mapmove.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>29</width>
           <height>29</height>
          </size>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="add">
         <property name="geometry">
          <rect>
           <x>890</x>
           <y>40</y>
           <width>41</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/res/add.png</normaloff>:/res/add.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>31</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="del">
         <property name="geometry">
          <rect>
           <x>950</x>
           <y>40</y>
           <width>41</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/res/del.png</normaloff>:/res/del.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>38</width>
           <height>38</height>
          </size>
         </property>
        </widget>
        <widget class="QPushButton" name="clamp">
         <property name="geometry">
          <rect>
           <x>1010</x>
           <y>39</y>
           <width>41</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/res/clamp.png</normaloff>:/res/clamp.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
        <widget class="QPushButton" name="nav">
         <property name="geometry">
          <rect>
           <x>1070</x>
           <y>39</y>
           <width>41</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/res/nav.png</normaloff>:/res/nav.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
        <widget class="QPushButton" name="ip_button">
         <property name="geometry">
          <rect>
           <x>770</x>
           <y>40</y>
           <width>41</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/res/ip.png</normaloff>:/res/ip.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>38</width>
           <height>38</height>
          </size>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>模型文件</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1767</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
